### DISCUSSION TITLE
How does trino support unsigned int64, if you query mysql's unsigned int64 will it overflow

### DISCUSSION THREAD

**Discussion by china-shang** [trinodb/trino#14679] [Q&A] [answered]
io.trino.spi.type.LongDecimalType  Will it be used to support this case ?

**Comment by tangjiangling** [comment, marked as answer]
> io.trino.spi.type.LongDecimalType Will it be used to support this case ?

Yes, when reading a MySQL `unsigned bigint` column, Trino converts it to `Decimal(20, 0)` (which internally in Trino is represented by `LongDecimalType`).
